File: server-data/resources/[halon]/utilities/ai.lua
Citizen.CreateThread(
    function()
        local multiplier = 0
        while true do
            Citizen.Wait(0)
            SetAmbientVehicleRangeMultiplierThisFrame(multiplier)
            SetCreateRandomCops(false)
            SetCreateRandomCopsNotOnScenarios(false)
            SetCreateRandomCopsOnScenarios(false)
            SetGarbageTrucks(false)
            SetParkedVehicleDensityMultiplierThisFrame(multiplier)
            SetPedDensityMultiplierThisFrame(multiplier)
            SetRandomBoats(false)
            SetRandomVehicleDensityMultiplierThisFrame(multiplier)
            SetScenarioPedDensityMultiplierThisFrame(multiplier, multiplier)
            SetVehicleDensityMultiplierThisFrame(multiplier)

            local player, ped = PlayerId(), PlayerPedId()
            local x, y, z, h = table.unpack(vector4(GetEntityCoords(ped), GetEntityHeading(ped)))

            ClearAreaOfCops(x, y, z, 100, 0)
            ClearPlayerWantedLevel(player)
            SetPoliceIgnorePlayer(player, true)
            SetPoliceRadarBlips(false)
        end
    end
)

File: ts/src/resources/base/client/ai.ts
import { Delay } from "@base/shared/main";
import Config from "@base/shared/config";

const { garbagetrucks, multiplier, randomboats, randomcops } = Config["ai"],
  multiplier_funcs = [
    SetAmbientVehicleRangeMultiplierThisFrame,
    SetParkedVehicleDensityMultiplierThisFrame,
    SetPedDensityMultiplierThisFrame,
    SetRandomVehicleDensityMultiplierThisFrame,
    SetScenarioPedDensityMultiplierThisFrame,
    SetVehicleDensityMultiplierThisFrame,
  ];

setTick(async function () {
  await Delay();
  SetGarbageTrucks(garbagetrucks);
  //for (const key in multiplier_funcs)
  //multiplier_funcs[key](multiplier, multiplier);
  SetRandomBoats(randomboats);

  const player = PlayerId(),
    [x, y, z] = GetEntityCoords(PlayerPedId());

  ClearAreaOfCops(x, y, z, 100, 0);
  ClearPlayerWantedLevel(player);
  SetPoliceIgnorePlayer(player, randomcops);
  SetPoliceRadarBlips(randomcops);
});
